addons/account/models/account_payment.py

```diff
@@ -159,6 +159,10 @@ class AccountPayment(models.Model):
         help='Negative value of amount field if payment_type is outbound')
     amount_company_currency_signed = fields.Monetary(
         currency_field='company_currency_id', compute='_compute_amount_company_currency_signed', store=True)
+    # used to get and display duplicate payment warning if partner, amount and date match existing payments
+    duplicated_payment_ids = fields.Many2many(comodel_name='account.payment', compute='_compute_duplicated_payment_ids')
+    # used to display duplicate payments, type integer to allow identifying only one duplicate or more
+    has_duplicated_payments = fields.Integer(compute='_compute_duplicated_payment_ids')
```

Comment:
I think that if we try to use the x2many_buttons, this field will be useless ? Also for the comments, the python convention is to put two spaces after the #

Reply:
Forgot to ask yesterday, but in the end I didn't change the comment because within that file the comments had only one space, so I thought it would look a bit off compared to the rest of the file. But if really should be changed, I can do that :)